test/Engine/FFITarget/DisposeTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace FFITarget
+{
+    public class DisposeTestClassA : IDisposable
+    {
+        static public int count { get; set; }
+
```

Comment:
Please add a static Reset method on these classes so that we can ensure that the tests don't interfere with each other.